### DISCUSSION TITLE
Seeding database with random data

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#8953] [Ideas]
It would be great if we had support for seeding database with random data. Mostly just generating elements with random field values. 
I know that  [seeder](https://plugins.craftcms.com/seeder) plugin exists, but for such potentially vulnerable functionality, i would much prefer a first-party plugin or even something built into the core.

**Comment by patmoshosting** [comment]
Has this been upvoted. Would be a great feature. Having a hard time using codeception to reseed the database.

If no action will be done on this, how can I go about seeding a database for testing? Its pretty unclear in the documentation.

**Reply by brandonkelly** [reply]
Craft doesn’t currently offer any built-in way to seed data, which is why it’s not mentioned in the docs.
